gateway/reverse_proxy_test.go

```diff
@@ -1498,7 +1498,7 @@ func TestGraphQL_OptionsPassThrough(t *testing.T) {
 			Code:    http.StatusOK,
 			HeadersMatch: map[string]string{
 				"Access-Control-Allow-Methods": http.MethodPost,
-				"Access-Control-Allow-Headers": "Content-Type",
```

Comment:
do we need this change as part of dependency update?

Reply:
yes, upstream modified behaviour to implement spec, enforces lowecase, signed off with patric